scripts/check-ts.js

```diff
@@ -0,0 +1,82 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+const childProcess = require("child_process");
+
+const event = process.env.GITHUB_EVENT;
+
+const body = JSON.parse(event);
+const baseSHA = body.pull_request.base.sha;
+const headSHA = body.pull_request.head.sha;
+
+const gitLogFilesCommand = `git diff --name-only ${baseSHA}...${headSHA}`;
```

Comment:
This will also return the paths of any file _deleted_ by the commit and thus do not exist anymore. `git diff --name-only --diff-filter=d` would filter them.

Reply:
Technically since I'm filtering the lines to only consider those starting with `+` (line 45) I don't have this issue but I could improve things here directly 👍